plugin/trino-hive/src/main/java/io/trino/plugin/hive/CachingDirectoryLister.java

```diff
@@ -181,6 +238,10 @@ public LocatedFileStatus next()
         };
     }
 
+    public Duration getFileListingTimeout() {
+        return fileListingTimeout;
```

Comment:
When rebasing my branch with the configuration against your branch I will update this value to be the configurable fileListingTimeout :) Great!